{"name": "test_login[test_data4]", "status": "passed", "attachments": [{"name": "log", "source": "b662b1d5-4847-42ef-8f65-cdc0036dd1f9-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "72829b29-1fb0-4920-bb5d-ef6710136160-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "test_data", "value": "{'case_id': 5, 'interface': 'login', 'title': '错误用户名', 'method': 'post', 'url': '/api/OAuth/Token', 'data': \"{\\n  'grant_type': 'password',\\n  'username': 'testtest',\\n  'password': '123456',\\n  'uuid': 690578652,\\n}\", 'expected': \"{'code': -1, 'msg': '用户名错误'}\", 'actual': \"{'code': -1, 'msg': '用户名错误', 'result': '2'}\", 'result': '通过'}"}], "start": 1613974147838, "stop": 1613974148187, "uuid": "58aae75e-35a6-472a-97ae-1f304ea93f3b", "historyId": "dbb92fe366a08e5391877711bcc252ac", "testCaseId": "2116f0ad00ab1de0effc54f114dd9496", "fullName": "tests.test_login.TestLogin#test_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-8M7P7VT"}, {"name": "thread", "value": "13412-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}